### DISCUSSION TITLE
[QUESTION] How to load saved data after change urlStore, urlLoad?

### DISCUSSION THREAD

**Discussion by nhan-tbt** [GrapesJS/grapesjs#4198] [Q&A]
Hi,
I trying to make multiple pages but i have a problem when i want to switch page by combobox.
My idea is changing urlStore, urlLoad of StorageManager with this code:

    editor.StorageManager.get('remote').set({
        urlStore: `${process.env.REACT_APP_STORAGE_API_URL}2`, 
        urlLoad: `${process.env.REACT_APP_STORAGE_API_URL}2`
    });

But data in canvas doesn't change.
Is there any way to load data in new URL after change Storage URL?

Please help me!
Thanks you!

**Comment by artf** [comment]
Multiple pages are part of the same project JSON, you don't need to change the storage options on page change.
